spec/index.html

```diff
@@ -5669,248 +5731,213 @@ <h5>NOT IN</h5>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, 2.0)</code></td>
+                    <td>false</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, "2"^^xsd:integer)</code></td>
+                    <td>true</td>
+                  </tr>
+                  <tr>
+                    <td><code>sameTerm(2, "02"^^xsd:integer)</code></td>
+                    <td>false</td>
+                  </tr>
+              </table>
+            </div>
+
+          </section>
+          <section id="func-sameValue">
+            <span id="func-RDFterm-equal"><!-- obsolete id --></span>
+            <h5>sameValue</h5>
+            <pre class="prototype nohighlight">
+              <span class="return">xsd:boolean</span> <span class="operator" style="text-transform: none;">sameValue</span> (<span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term1</span>, <span class="type"><span class="type RDFterm">RDF term</span></span> <span class="name">term2</span>)
+            </pre>
+
+            <p>This function cannot be used directly in expressions. The purpose
+              of this function is to define the semantics of the "=" operator when applied to
+              two RDF terms that do not fall into any of the other, more concrete cases
+              covered in the operator mapping table in Section
+              <a href="#OperatorMapping" class="sectionRef"></a>.
+            </p>
+
+            <p>The result of this function is determined by going through the following steps.</p>
+
+            <ol>
+              <li>If <code>term1</code> and <code>term2</code> are
+                <a data-cite="RDF12-CONCEPTS#dfn-rdf-term-equality">equal RDF terms</a>,
+                then return TRUE.
+              </li>
+              <li>If <code>term1</code> and <code>term2</code> are both
+                <a data-cite="RDF12-CONCEPTS#dfn-literal">literals</a>
+                and one or both of these literals are known to be
+                <a data-cite="RDF12-CONCEPTS#dfn-ill-typed">ill-typed</a>,
+                then produce a type error.
               </li>
               <li>
-                <span class="name">term1</span> is a <span class="literal type">literal</span> and
-                <span class="name">term2</span> is a <span class="literal type">literal</span> such that
-                these two <span class="literal type">literals</span> are equal as per the notion of <a data-cite="RDF12-CONCEPTS#dfn-literal-term-equality">Literal term equality</a> of
-                [[RDF12-CONCEPTS]].
+                "NaN"^^xsd:double and "NaN"^^xsd:float are considered to 
+                represent the same value.
+                If <code>term1</code> and <code>term2</code> are
+                both "NaN" for either xsd:double or xsd:float then 
+                return TRUE.
```

Comment:
This issue with specifying this here is that if we consider that if the "operator mapping table" has precedence to this definition then `"NaN"^^xsd:double = "NaN"^^xsd:float` must be evaluated according to [op:numeric-equal](https://www.w3.org/TR/xpath-functions-31/#func-numeric-equal) and so `"NaN"^^xsd:double = "NaN"^^xsd:float`  returns FALSE, opposite to this definition.

Ammending instead the "operator mapping table" to state we follow [op:numeric-equal](https://www.w3.org/TR/xpath-functions-31/#func-numeric-equal)  except that NaN = NaN might be better. Or we can ignore this case and have it implementation defined.